### PR TITLE
GraphQL mutation for creating an aid request

### PR DESCRIPTION
**Pull request by bluquar**
https://user-images.githubusercontent.com/4309735/146652901-0412e9a9-4c7e-4a7d-be3a-0d542c18c007.mov

### CODE REVIEW

**Comment by lowellbander** [comment]
LOL i take great joy from the personal-ness of this test plan <3

**Comment by lowellbander** [comment]
haven't read code yet, but consider validating in test plan that request cannot be created if user is not logged in

**Comment by lowellbander** [comment]
Will you please grant me access to the MongoDB cluster so I can peruse our DB?

File: src/server/aid_requests/index.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { model, Schema } from 'mongoose';
+
+export const GQL_AID_REQUEST_TYPES = `
```
Comment (lowellbander) [inline src/server/aid_requests/index.ts:3]: what do you think about trying to automate some of this with https://github.com/graphql-compose/graphql-compose-mongoose ?

File: src/server/aid_requests/index.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { model, Schema } from 'mongoose';
```
Comment (lowellbander) [inline src/server/aid_requests/index.ts:1]: I remember you saying previously you wanted to use Mongoose just for Users, but then use MongoDB's Node.JS driver for other stuff -- why the change of heart?

**Review by lowellbander — APPROVED** [review]
it's happeninggggg

File: src/server/aid_requests/index.ts (hunk 2 above)
Comment (bluquar) [reply src/server/aid_requests/index.ts:1]: Basically just because mongoose was the first thing that came up when I searched for a guide

File: src/server/aid_requests/index.ts (hunk 1 above)
Comment (bluquar) [reply src/server/aid_requests/index.ts:3]: That seems cool. Can you make an issue?

Comment (lowellbander) [reply src/server/aid_requests/index.ts:3]: ofc!

Comment (lowellbander) [reply src/server/aid_requests/index.ts:3]: #54

**Comment by bluquar** [comment]
> haven't read code yet, but consider validating in test plan that request cannot be created if user is not logged in

sent you an invite to join my mongo project
